translations/ui-orders/en.json

```diff
@@ -715,15 +697,34 @@
   "settings.closingReasons.addBtn": "Add",
   "settings.closingReasons.cancelBtn": "Cancel",
   "settings.closingReasons.editBtn": "Edit",
+  "settings.closingReasons.reason.Canceled": "Canceled",
```

Comment:
Can we leave the previous translations? IMHO, it's just a key renaming, looks like over-engineering

Reply:
I removed default closing reasons and translate only reasons from BE with the source `System`. Now translations keys = reason from BE

Reply:
does it makes sense? What if guys on the back-end change one letter? IMHO, it's not bulletproof solution at all. To me it looks like the transition solution should look like a map of keys - from back-end to the UI translations. Just to reduce unnecessary work on the translations side.

Reply:
Ok, reverted translations